day-1/kane-solution.js

```diff
@@ -0,0 +1,12 @@
+var sum = 0
+
+var summation = function () {
```

Comment:
summation is an awkward name. what is this a summation of?

Maybe sumMultiples3and5To(var limit) would make more sense?